src/components/Analytics/PageviewTracker.tsx

```diff
@@ -3,6 +3,18 @@ import { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
 import ReactGA from 'react-ga';
 import { defaultTracker, legacyTracker } from './utils';
+import amplitude from 'amplitude-js';
+
+const AMPLITUDE_KEY_DEV = '9273bc15ce71641291d471c9f17895a5';
+const AMPLITUDE_KEY_PROD = 'c92804b9b1f5323200e94002a76a86a9';
+
+function initializeAmplitude() {
+  const amplitudeKey =
+    process.env.NODE_ENV === 'production'
```

Comment:
Hrm...  we have an existing `getEnvironment()` function that uses `process.env.REACT_APP_ENVIRONMENT`... Not 100% sure if they behave the same, but let's reuse that if we can.

https://github.com/covid-projections/covid-projections/blob/6add9e54681a519c44af9b1c72a4b98595a81a9c/src/common/utils/environment.ts#L8